### PR TITLE
Fix UI tablet issues mobile

### PR DESCRIPTION
**Pull request by hoangtranlehuy**
Mobile App: fix issues UI Tablet
Fix UI Tablet BottomSheetModal, Modal, Home UI, MyQRcode, Invite Modal, Add role channel setting, User status tablet, DM message screen, remove bottom navigation
https://github.com/nccasia/mezon/issues/4908

### CODE REVIEW

File: apps/mobile/src/app/componentUI/MezonConfirm/styles.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ export const style = (colors: Attributes) =>
 			padding: Metrics.size.xl,
 			margin: Metrics.size.l,
 			borderRadius: 10,
-			overflow: 'hidden'
+			overflow: 'hidden',
+			marginHorizontal: '30%'
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/componentUI/MezonConfirm/styles.ts]: check mobile

File: apps/mobile/src/app/screens/home/homedrawer/DrawerContent.tsx [hunk 2]
```diff
@@ -64,9 +65,14 @@ const DrawerContent = React.memo(() => {
...
-			{!isTabletLandscape && <BackNativeListener />}
-			<ChannelListWrapper />
+			<View style={styles.container}>
+				<View style={styles.rowContainer}>
+					<ServerList />
+					<BackNativeListener />
+					<ChannelListWrapper />
+				</View>
+				{isTabletLandscape && <ProfileBar />}
+			</View>
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/DrawerContent.tsx:75]: recheck on mobile

**Comment by hoangtranlehuy** [comment]
Screenshot 2024-11-19 170537
Screenshot 2024-11-19 170557
Screenshot 2024-11-19 170618
Screenshot 2024-11-19 170757
Screenshot 2024-11-19 170914

File: apps/mobile/src/app/components/Event/EventViewer.tsx [hunk 3]
```diff
@@ -70,6 +70,7 @@ export function EventViewer({ handlePressEventCreate }: { handlePressEventCreate
 				<MezonTab
 					views={[<EventDetail event={currentEvent} eventDetailRef={bottomSheetDetail} />, <EventMember event={currentEvent} />]}
 					titles={['Event Info', 'Interested']}
+					isBottomSheet={true}
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/Event/EventViewer.tsx]: sao cái này hardcode = true?

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**